ui/app/templates/components/tooltip.hbs

```diff
@@ -3,6 +3,10 @@
   SPDX-License-Identifier: MPL-2.0
 ~}}
 
-<span class="tooltip" aria-label="{{this.text}}">
+{{#if this.condition}}
```

Comment:
Request: could we add some tests for condition? Is it expected to always be a boolean? If so, let's consider adding an assert/check to make sure of its type.

In particular I worry about a condition that results in a falsy value being passed in, like `@condition=job.version` where version could be 0 / result in a failed condition.

Reply:
We can design the component API to have an assertion where only booleans can be passed here to prevent this. That's probably the most appropriate solution IMO. Are you OK with that?